src/pipelines/mod.rs

```diff
@@ -34,7 +34,8 @@ const ATTR_REL: &str = "rel";
 const SNIPPETS_DIR: &str = "snippets";
 const TRUNK_ID: &str = "data-trunk-id";
 
-/// A model of all of the supported Trunk asset links expressed in the source HTML as `<trunk-link/>` elements.
+/// A model of all of the supported Trunk asset links expressed in the source HTML as
```

Comment:
We'll need to get this checked with @thedodd for these formatting changes.

Reply:
Any changes to formatting were done by `cargo fmt`.